LabExT/View/MeasurementControlSettings.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2021  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
+"""
+import json
+import logging
+import os.path
+from tkinter import Toplevel, Checkbutton, BooleanVar, Label, StringVar, Entry, Button
+from tktooltip import ToolTip
+
+from LabExT.Utils import get_configuration_file_path
+from LabExT.View.Controls.CustomFrame import CustomFrame
+
+
+class MeasurementControlSettings:
```

Comment:
very cool concept of having this "settings" class with a store/load method built in!